control-plane-operator/controllers/hostedcontrolplane/assets/cluster-machine-approver/cluster-machine-approver-deployment.yaml

```diff
@@ -0,0 +1,67 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: machine-approver
+  labels:
+    app: machine-approver
+    machine-approver: "true"
+spec:
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      app: machine-approver
+  template:
+    metadata:
+      name: machine-approver
+      annotations:
+        target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
+      labels:
+        app: machine-approver
+    spec:
+      serviceAccountName: machine-approver-sa
+      containers:
+      - name: machine-approver-controller
+{{- if atleast_version "4.10.0" }}
+        image: {{ imageFor "cluster-machine-approver" }}
+{{- else }}
+        image: quay.io/hanqiuzh/origin-cluster-machine-approver@sha256:ae555a2d6bbf99ea5ed0764216f06c759ac0fd861340afda8f714a29bd2da44b
```

Comment:
cannot find an latest stable image we can reference, so used one built from source